app/views/tournaments/show.html.erb

```diff
@@ -1,4 +1,12 @@
 <div class="container">
+  <h1 class="mb-3 border-bottom border-3"><%= @tournament.event.location %></h1>
```

Comment:
Do we need all this stuff added back to the show page?

Reply:
Seems okay, we need to display the location somewhere

Reply:
It's already displayed in the h1 that was already there

Reply:
ah true